augur/augur_client.py

```diff
@@ -180,13 +180,16 @@ def filter_blocks(self, start_block, end_block, filter, handler,
       IOError: If there is an issue with communicating with the node.
     '''
     self._require_is_open()
-    if (start_block < 0) or (end_block < 0) or (increment < 0):
-      raise IOError('Block numbers or increment cannot be less than 0.')
+    if start_block < 0:
+      raise IOError('start_block cannot be less than 0.')
+    if end_block < 0:
```

Comment:
This check is redundant, it is a strictly weaker check than the one performed on line 189.